From: Ole Streicher <ole@aip.de>
Date: Tue, 12 May 2020 14:18:07 +0200
Subject: Don't install lsm2bin

This seems not an end-user script, and we don't have a manpage for it
---
 setup.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.py b/setup.py
index b30171b..627be5d 100644
--- a/setup.py
+++ b/setup.py
@@ -108,9 +108,6 @@ setup(
     entry_points={
         'console_scripts': [
             'tifffile = tifffile:main',
-            'tiffcomment = tifffile.tiffcomment:main',
-            'tiff2fsspec = tifffile.tiff2fsspec:main',
-            'lsm2bin = tifffile.lsm2bin:main',
         ],
         # 'napari.plugin': ['tifffile = tifffile.napari_tifffile'],
     },
